SAR_Training/English/Master/mRFDIvsNDVI.ipynb

```diff
@@ -23,6 +23,40 @@
     "- How to compare RFID and Landsat-8-derived NDVI maps"
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    env = !echo $CONDA_PREFIX
</pre><p>I think it would be more pythonic to use os.getenv("CONDA_PREFIX") here (and in the line above)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ASFOpenSARlab/opensarlab-notebooks/pull/205/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>